demo/src/app/components/dropdown/demos/basic/dropdown-basic.html

```diff
@@ -0,0 +1,8 @@
+ <div ngbDropdown [open]="true">
```

Comment:
Do we want to have it open by default in the simplest possible demo? I would drop `[open]="true"` in the basic demo - we can add another demo in the future to show-case this attribute.